MetaMask/metamask-extension
# Enable "Add Popular Network" feature by default for all users

Fixes: https://github.com/MetaMask/metamask-extension/issues/16016

## More Information

- Removes code related to enabling/disabling the "Add Popular Network" feature
- Persists the feature for all users
- Removes "What's New" notification which prompted users to turn on the setting
- Removes the Experimental toggle for "Add Popular Network"

## Manual Testing Steps
1. Ensure all paths to adding a network show the Popular networks that can be added
2. Ensure the Experimental toggle is no longer available
3. Ensure the old "What's New" notification does not show

## Pre-Merge Checklist


### + If there are functional changes:
